velox/dwio/common/tests/utils/DataFiles.cpp

```diff
@@ -26,12 +28,19 @@ bool endsWith(const std::string& s, const std::string& suffix) {
 } // namespace
 
 std::string getDataFilePath(
-    const std::string& baseDir,
+    const std::string& pathFromFbcode,
```

Comment:
Maybe we don't want to expose this name

Reply:
@Yuhta Jimmy, thanks for reviewing the PR. Did you mean use another name that does not include "fb"? The code is like the following.
```
std::string getDataFilePath(
    const std::string& pathFromFbcode,
    const std::string& filePath) {
  std::string current_path = fs::current_path().c_str();
  if (endsWith(current_path, "fbcode")) {
    return current_path + "/" + pathFromFbcode + "/" + filePath;
  }
  return current_path + "/" + filePath;
}
```
If "fb" is a concern here, it did already include `endsWith(current_path, "fbcode")`. I actually don't know why this function was this way, but the baseDir was very confusing, since it is not used in my dev environement at all, and baseDir makes people think the path is baseDir + filePath but it's actually not. I created the function below that doesn't have this argument. Adn the renaming of pathFromFbcode is just to remove confusions. Do you have a recommendation what we should use here?

Reply:
I see, this is a hack someone has done to make it run with buck.  I was concerning about the name `fbcode`, but since it already appears in the code, it's ok to leave it like that.